gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/GravatarQuickEditorPage.kt

```diff
@@ -48,6 +49,12 @@ internal fun GravatarQuickEditorPage(
 ) {
     val navController = rememberNavController()
 
+    LaunchedEffect(confirmDismissal) {
+        if (confirmDismissal && navController.currentDestination?.route != EditorNavDestinations.QUICK_EDITOR.name) {
+            onDoneClicked()
+        }
+    }
```

Comment:
Now that we have to allow for sheet dismissal, each destination in this file needs to handle this separately. 
To keep it simple, I kept the logic here, and only the `EditorNavDestinations.QUICK_EDITOR` will have to handle it individually, since all the other pages confirm the dismissal.